lib/view/screen/home_screen.dart

```diff
@@ -77,7 +77,7 @@ class HomePageTemplate extends TemplateScreen {
                   size: 38,
                 ),
                 onPressed: () {
-                  Routes.pushNamed(Routes.SCREEN_ADD_BUDGET,
+                  Routes.pushNamed(Routes.SCREEN_ADD_TRANSACTION,
```

Comment:
Don't remove route for add budget in home screen. Instead, transfer route for add transaction in transaction screen add button.